### DISCUSSION TITLE
2024-04-24

### DISCUSSION THREAD

**Discussion by jeasonstudio** [ant-design/ant-design-web3#822] [中文社区双周会]
上次双周会记录：https://github.com/ant-design/ant-design-web3/discussions/783

**Comment by jeasonstudio** [comment]
issue：
* https://github.com/ant-design/ant-design-web3/issues/815 讨论下在一个包里兼容 ethers 两个版本的方式
* https://github.com/ant-design/ant-design-web3/issues/814 图标的分类原则
* https://github.com/ant-design/ant-design-web3/issues/800 rune 热度有哪些可以沾边的部分

pr：
* https://github.com/ant-design/ant-design-web3/pull/812 收款组件进展
* https://github.com/ant-design/ant-design-web3/pull/786 solana wallets
* https://github.com/ant-design/ant-design-web3/pull/772 web3js adapter 进展

待设计稿：
* connect-button 签名状态
* wallect-connect 改名、改图标

**Comment by gin-lsl** [comment]
把 Example 单独创建一个 package，避免有些 demo 需要用到一些 @ant-design/web3 不需要的依赖。
